parser.go

```diff
@@ -47,7 +47,7 @@ func ParseMetric(packet []byte) (*Metric, error) {
 		return nil, errors.New("Invalid metric packet, need at least 1 pipe for type")
 	}
 	// Now convert it
-	v, err := strconv.ParseInt(string(data[0]), 10, 32)
+	v, err := strconv.ParseFloat(string(data[0]), 64)
 	if err != nil {
 		return nil, fmt.Errorf("Invalid integer for metric value: %s", parts[1])
```

Comment:
whoopsies, missed this on the first pass, should probably say "float" or "number" now

although I'm going to be changing this error condition anyway, so we can leave it if you prefer